api/src/main/java/io/strimzi/api/kafka/model/KafkaExporterResources.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+/**
+ * Encapsulates the naming scheme used for the resources which the Cluster Operator manages for a
+ * {@code KafkaExporter} cluster.
+ */
+public class KafkaExporterResources {
+    protected KafkaExporterResources() { }
+
+    /**
+     * Returns the name of the Kafka Exporter {@code Deployment} for a {@code KafkaExporter} cluster of the given name.
+     * @param clusterName  The {@code metadata.name} of the {@code KafkaExporter} resource.
+     * @return The name of the corresponding Kafka Exporter {@code Deployment}.
+     */
+    public static String deploymentName(String clusterName) {
```

Comment:
I think we are abusing of the "cluster" term. I remember I saw this on other resources I guess. Can we talk about a `KafkaExporter` cluster here? `clusterName` as described by the Javadoc is the name of the resource while its name let me think about the Kafka cluster for which I am going to deploy the Kafka Exporter (while it's referring to the "cluster" Kafka Exporter). Wdyt @tombentley @scholzj ?

Reply:
There is no KafkaExporter cluster or it's name. I think this *is* the name of the `Kafka` resource here, or? In which case I think this is consistet with our other usecases everywhere else.

Reply:
Ok ... I was on vacation for too long. Of course, the Kafka Exporter doesn't have its own CRD so that it has a "name" per se ... it's obviously the Kafka cluster name ... sorry for the noise.

Reply:
No worries. I agree this could be confusing sometimes what name is what. But if then it would be probably for a different PR.